torch/autograd/gradcheck.py

```diff
@@ -725,47 +716,65 @@ def _differentiable_outputs(x):
     return tuple(o for o in _as_tuple(x) if o.requires_grad)
 
 
-def get_notallclose_msg(analytical, numerical, output_idx, input_idx, error_str='') -> str:
-    return error_str + 'Jacobian mismatch for output %d with respect to input %d,\n' \
+def get_notallclose_msg(analytical, numerical, output_idx, input_idx, complex_indices, inp_is_complex) -> str:
+    out_is_imag = complex_indices and output_idx in complex_indices
+    grad_output = "1j" if out_is_imag else "1"
+    prefix = "" if not inp_is_complex and not out_is_imag else \
+        f"Gradients failed to compare equal for grad output = {grad_output}"
```

Comment:
Well, we shouldn't talk about grad_output here anymore.
Maybe only "while considering the real part of the complex outputs only"